package.json

```diff
@@ -8,20 +8,26 @@
     "start": "next start",
     "lint:js": "next lint",
     "lint:css": "stylelint '**/*.scss'",
-    "lint": "run-s lint:js lint:css"
+    "lint": "run-s lint:js lint:css",
+    "test": "jest", 
+    "test:watch": "npm run test -- --watchAll"
```

Comment:
🏆  Love this! This is great! What do you think about adding to the `README.md` so people know about how to run tests and what the difference is between the two?

Most people would recognize it but this is a good exercise in updating documentation.

Reply:
I think we should definitely add it to the docs. Should I add instructions similarly to the PR template?

Reply:
Hmm, it looks like it might already be in there under validation ("Confirm all tests pass."). Were you thinking about making it more specific?

Reply:
Oh that's a good idea. What do you think of this? 

## Running Jest Tests
There are two ways to run tests on your code: 
1. `npm run test`: runs a single test command. Get feedback one time. 
2. `npm run test:watch`: constantly looks for changes to your files and reruns tests. Get continuous feedback.

Reply:
Love that! Maybe even go further and put [that in a table](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables)? That way, you can add more as the code evolves in a way that is still readable. It'd look something like this.

<img width="741" alt="Screen Shot 2022-10-06 at 5 21 51 PM" src="https://user-images.githubusercontent.com/7015466/194420968-064eaae1-afef-4c3d-a897-6c72d7c0c7d2.png">